### ISSUE TITLE
Does not work under Windows

### ISSUE THREAD

**Issue by vekunz** [mbolotov/intellij-playwright#13]
This plugin does not work if I try to use it under Windows. It creates this command:
```
"C:\Program Files\nodejs\npx.cmd" playwright test --reporter C:\Users\xxxx\AppData\Local\Temp\intellij-playwright-reporter13237922888003142249.js tests\create-folders.spec.ts
```

The problem is the `tests\create-folders.spec.ts`. The plugin must replace the `\` with a `/` (or with `\\`, but just a `/` should be better).

**Comment by vekunz** [comment]
Hi, I have the version 1.1.2 of the plugin with WebStorm 2022.2.1. I use Node.js 16.17.0 (latest LTS). Playwright is version 1.25.1 (should be latest too).

If I execute a file, I get this this console output:
```
"C:\Program Files\nodejs\npx.cmd" playwright test --reporter C:\Users\xxxx\AppData\Local\Temp\intellij-playwright-reporter13237922888003142249.js tests\create-folders.spec.ts

Running 0 tests using 0 workers

=================
 no tests found.
=================
```

If I copy the command in a cmd, I get the same error, but if I switch the `\` to a `/`, it works.

**Comment by mbolotov** [comment]
Can you try this EAP plugin bundle?
I've added the logic to switch the slash to `/`
[Maestro-1.2-EAP2.zip](https://github.com/mbolotov/intellij-playwright/files/9498274/Maestro-1.2-EAP2.zip)

**Comment by vekunz** [comment]
With this, everything works fine.

**Comment by Doug-Bowen** [comment]
This happens for me as well.

**Comment by mbolotov** [comment]
@Doug-Bowen, does the [Maestro-1.2-EAP2](https://github.com/mbolotov/intellij-playwright/files/9498274/Maestro-1.2-EAP2.zip) bundle works for you?

I'm planning to release a new version of the plugin including the fix for this issue in several days.

**Comment by Doug-Bowen** [comment]
Attempting to get the EAP bundle installed but struggling.  Any idea where this should be stored?  I'm on Windows.

**Comment by mbolotov** [comment]
1. download it to any accessible place in the file system. 
2. install the plugin from the System settings:
![image](https://user-images.githubusercontent.com/19785715/190217544-595f18db-c265-48e9-9dd0-8bd587e77a50.png)

here the official docs: https://www.jetbrains.com/help/idea/managing-plugins.html#install_plugin_from_disk

**Comment by Doug-Bowen** [comment]
got it.  works flawlessly

**Comment by mbolotov** [comment]
fixed in 1.2 version
